modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -309,7 +304,7 @@ Though existing properties can be re-set using this procedure to minimize store
 [[indexes-vector-similarity]]
 == Supported similarity functions
 
-The choice of similarity function can vastly differ in which indexed vectors are considered similar, and which are valid.
+The choice of similarity function can vastly differ depending on which indexed vectors are considered similar, and which are valid.
```

Comment:
This still reads the wrong way around to me.
So I think we can change the line to something like:
```suggestion
The choice of similarity function affects which indexed vectors are considered similar, and which are valid.
```